Atomic/backends/_docker.py

```diff
@@ -338,12 +338,13 @@ def pull_image(self, image, remote_image_obj, **kwargs):
         if '@sha256:' in image:
             image = image.replace("@sha256:", ":")
 
+        src_creds = kwargs.get('src_creds')
         insecure = True if util.is_insecure_registry(self.d.info()['RegistryConfig'], registry) else False
         trust = Trust()
         trust.discover_sigstore(fq_name)
         util.write_out("Pulling {} ...".format(fq_name))
         util.skopeo_copy("docker://{}".format(fq_name), image, debug=debug, insecure=insecure,
-                         policy_filename=trust.policy_filename)
+                         policy_filename=trust.policy_filename, src_creds=src_creds)
```

Comment:
Should you (or are you planning to later) add the destination creds too?

Reply:
with the storage types we support in atomic we don't write to any destination registry, so I think that is not necessary yet.  Or is there any case where skopeo_copy is used with a destination registry?

Reply:
@giuseppe It's possible for skopeo to copy to a destination registry, how much it's used in practice I don't know.  I'd guess it's not a common occurrence and probably less likely in the Atomic arena.  @mtrmac any thoughts?

Reply:
yes, Skopeo copies to a destination registry but at the moment, AFAICS, this possibility is not plugged into `atomic pull`.  Until we have that supported, it makes no sense to expose dest-creds.

Reply:
something→registry is called `push` in `atomic`; in this case, the destination (`image`) is always `docker-daemon:`, which does not need credentials (of this kind at least; it requires access to the socket).